src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,155 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.InvalidDataException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static final int MIN_AGE = 18;
+    private RegistrationServiceImpl registrationService;
+
+    @BeforeEach
+    void setup() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
```

Comment:
seems you forgot to clear the state between tests

think of `@AfterEach` and what should be inside that method